jupyterhub/tests/test_scopes.py

```diff
@@ -262,79 +260,121 @@ async def test_by_fake_user(app):
 err_message = "No access to resources or resources not found"
 
 
-async def test_request_fake_user(app):
-    user_name = 'buster'
-    fake_user = 'annyong'
-    user = add_user(app.db, name=user_name)
-    test_role = generate_test_role(user_name, ['read:users!group=stuff'])
-    roles.create_role(app.db, test_role)
-    roles.grant_role(app.db, entity=user, rolename='test')
+@pytest.fixture
+def temp_role(app):
```

Comment:
Maybe call this `make_temp_role`? It's often nice for functions to have a verb name, while items have noun names. Looking at the tests, it's reasonable to expect `temp_role` fixture to *be* a temporary role, when it's really a function to make one.

Reply:
Agree! Taken care of